test/test-application/src/index.ts

```diff
@@ -54,13 +58,18 @@ async function testShowcase() {
     port: 7469,
     auth: fakeGoogleAuth,
   };
+  
 
   const grpcClient = new EchoClient(grpcClientOpts);
+  const grpcSequenceClient = new SequenceServiceClient(grpcClientOpts);
+
   const fallbackClient = new EchoClient(fallbackClientOpts);
   const restClient = new EchoClient(restClientOpts);
 
   // assuming gRPC server is started locally
-  await testEcho(grpcClient);
```

Comment:
`testEcho` is still useful, maybe keep it?

Reply:
Yes! This was a mistake. Good eyes